SimTracker/SiPhase2Digitizer/test/PixelTestBeamValidation.cc

```diff
@@ -0,0 +1,735 @@
+//
+// Access Digi collection and creates histograms accumulating
+// data from the module to different pixel cells, 1x1 cell,
+// 2x2 cell, etc..
+//
+// Author:  J.Duarte-Campderros (IFCA)
+// Created: 2019-10-02
+//
+
+// CMSSW Framework
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ESWatcher.h"
+
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
+
+// CMSSW Data formats
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/DetId/interface/DetId.h"
+
+// system
+#include <algorithm>
+
+// XXX - Be careful the relative position
+#include "PixelTestBeamValidation.h"
+
+const double unit_um = 1e-4;  // [cm]
+// NOT USED
+// const double unit_mm = 1e-1;  // [cm]
+
+const double unit_degree = 0.017453292519943295;  // [rad]
```

Comment:
instead of this, use DataFormats/Math/interface/angle_units.h ```convertRadToDeg``` or ```degPerRad```

Reply:
Ok, use _deg from CMSUnits